resources/css/theme.css

```
@@ -0,0 +1,9 @@
+@import 'tailwindcss';
+
+@import '../../../../../vendor/filament/filament/resources/css/theme.css';
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)